staging/src/k8s.io/kube-proxy/config/v1alpha1/types.go

```diff
@@ -201,6 +201,10 @@ type KubeProxyConfiguration struct {
 	EnableProfiling bool `json:"enableProfiling"`
 	// showHiddenMetricsForVersion is the version for which you want to show hidden metrics.
 	ShowHiddenMetricsForVersion string `json:"showHiddenMetricsForVersion"`
+	// The value for the "service.kubernetes.io/service-proxy-name" label that this
+	// kube-proxy instance shall handle. If unset (default), kube-proxy will handle
+	// any service that has NOT set this label.
+	ServiceProxyName string `json:"serviceProxyName,omitempty"`
```

Comment:
I will keep this as string, because the cli flag can't handle a *string in a good way. But and empty label ("") will be handled as a LabelSelector for a non-set label (nil).